packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -122,12 +181,12 @@ class SelectableTableRows extends React.Component {
 
   render() {
     const {
-      children, disableUnselectedRows, onChange, selectedIndexes, ...customProps
+      children, disableUnselectedRows, onChange, onBlur, selectedIndexes, ...customProps
     } = this.props;
     const clonedChildItems = this.clonedChildItems(children);
 
     return (
-      <TableRows {...customProps}>
+      <TableRows id={rowsSelector} {...customProps}>
```

Comment:
Please correct me if I am wrong - if the page has multiple tables, then are we adding the same id to the `TableRows` in all the tables?

Reply:
Base level components within Terra should rarely have IDs because there can generally be multiple on the same page. Those components that do have them generally require a unique identifier to be generated unless specific constraints enforce that only one is ever present on a single page. 

It should be possible to add the aria labels prior to rendering the table into the dom. Potentially within the `clonedChildItems` function.

Reply:
@StephenEsser  I tried adding aria-label attribute in clonedChildItems but there are no ref added to row to access dom element. it throws below warning when i try to add ref with [newProps](https://github.com/cerner/terra-core/blob/f4e469fc37e008087364c2d01dfcabee2fe1b83f/packages/terra-table/src/SelectableTableRows.jsx#L163)  ' Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()? ' it will be helpful to know if there are any other ways to add ARIA attribute to react constants or to access dom element from row prop in [newPropsForRow()](https://github.com/cerner/terra-core/blob/f4e469fc37e008087364c2d01dfcabee2fe1b83f/packages/terra-table/src/SelectableTableRows.jsx#L138) method

Reply:
adding aria-labels prior to rendering the table in [clonedChildItems](https://github.com/cerner/terra-core/blob/153a90093ca106ce109d4c4d0ab861ec611985b2/packages/terra-table/src/SelectableTableRows.jsx#L165).

replaced ID with data-attribute in [selected rows](https://github.com/cerner/terra-core/blob/153a90093ca106ce109d4c4d0ab861ec611985b2/packages/terra-table/src/SelectableTableRows.jsx#L181)

 @StephenEsser Thank you for suggesting